recipe/conda_build_config.yaml

```diff
@@ -179,18 +185,18 @@ pyqt:
 qt:
   - 5.6
 readline:
-  - 7
+  - 7.0
 r_base:
   - 3.3.2
   - 3.4.1
 scipy:
   - 0.19
 snappy:
-  - 1
+  - 1.1.6
 sox:
-  - 14.4
+  - 14.4.2
 sqlite:
-  - 3
+  - 3.20
```

Comment:
This is [our pinning]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L122 ), but not what we are using. Please see issue ( https://github.com/conda-forge/sqlite-feedstock/issues/25 ) for the publication issue. We may need to go to `3.21.0` to fix it.